exercises/nucleotide-count/HINTS.md

```diff
@@ -0,0 +1,3 @@
+##HINTS
+For this exercise the following C# feature comes in handy: 
```

Comment:
I made the changes above and resubmitted (did a new branch and followed the steps you list above so hopefully only the one change is present).